go/protocol/keybase1/tlf_keys.go

```diff
@@ -21,7 +21,8 @@ const (
 	TLFIdentifyBehavior_SALTPACK        TLFIdentifyBehavior = 7
 	TLFIdentifyBehavior_CLI             TLFIdentifyBehavior = 8
 	TLFIdentifyBehavior_GUI             TLFIdentifyBehavior = 9
-	TLFIdentifyBehavior_DEFAULT_KBFS    TLFIdentifyBehavior = 10
+	TLFIdentifyBehavior_PAGES           TLFIdentifyBehavior = 10
```

Comment:
Shouldn't we make this one be 11, rather than change an existing value?  Otherwise on Linux the core service can be restarted without restarting KBFS, and the values passed between them will be inconsistent.

Reply:
Good point; will do.